integration/kubernetes/k8s-nested-configmap-secret.bats

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2021 IBM Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	export KUBECONFIG="${KUBECONFIG:-$HOME/.kube/config}"
+	get_pod_config_dir
+
+	pod_name="nested-configmap-secret-pod"
+
+	# Creates a configmap, secret and pod that mounts the secret inside the configmap
+	kubectl create -f "${pod_config_dir}/pod-nested-configmap-secret.yaml"
```

Comment:
Normally, `setup` is only used to prepare test configs, the creation of pods should be in the test body(`@test`).

Reply:
Fair enough -- I was trying a pattern where setup does the opposite of teardown.